.github/CODEOWNERS

```
@@ -2,12 +2,12 @@
 * @elastic/elastic-agent-control-plane
 
 # Top-level files ownership
-/catalog-info.yaml @elastic/observablt-ci
+/catalog-info.yaml @elastic/elastic-agent-control-plane @elastic/observablt-ci
 
 # Sub-directories/files ownership. Remember that order matters; the last matching pattern takes the most precedence.
-/.buildkite @elastic/observablt-ci
-/.ci @elastic/observablt-ci
-/.github @elastic/observablt-ci
+/.buildkite @elastic/elastic-agent-control-plane @elastic/observablt-ci
+/.ci @elastic/elastic-agent-control-plane @elastic/observablt-ci
```

I'd say `/.ci/scripts/update-otel.sh` should be moved away from pinging `observablt-ci` (us)

```suggestion
/.ci @elastic/elastic-agent-control-plane @elastic/observablt-ci
/.ci/scripts/update-otel.sh @elastic/elastic-agent-control-plane
```